types/lsofi/tsconfig.json

```
@@ -1,16 +1,25 @@
 {
 	"compilerOptions": {
+		"esModuleInterop": true,
 		"module": "commonjs",
 		"noEmit": true,
 		"forceConsistentCasingInFileNames": true,
 		"baseUrl": "../",
-		"typeRoots": ["../"],
+		"typeRoots": [
+			"../"
+		],
 		"types": [],
-		"lib": ["ES6"],
+		"lib": [
+			"ES6",
+			"DOM"
```

```suggestion
			"ES6"
```
there is nothing from `lib.dom` used for this to be required, am I correct?

---

Yep, that was old from `http`.